### PR TITLE
Tooltip accessibility

### PR DESCRIPTION
**Pull request by YahelCh**

### CODE REVIEW

File: test/rules/accessibility/tooltip-accessibility.spec.js [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+    it('tooltip with no described element should result in an error', function () {
+        var code = `<div>
+                   ${tooltip}
+                    </div>`;
+        var messages = HTMLHint.verify(code, ruleOptions);
+        expect(messages.length).to.be(1);
+        expect(messages[0].rule.id).to.be(ruldId);
+        expect(messages[0].message).to.equal(formattedMessage(tooltip));
+        // expect(messages[0].line).to.be(3);
```
Comment (rziv) [inline test/rules/accessibility/tooltip-accessibility.spec.js]: uncomment and add the correct line/column numbers here and in the following test cases

File: src/rules/accessibility/tooltip-accessibility.js [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+    description: 'element with binding "addTooltip" should have binding "addDescription"',
+    init: function (parser, reporter) {
+        var self = this;
+        var looseTooltips = [];
+        var isTooltip = function (event) {
+            var tagName = event.tagName;
+            var hasTooltipClass = HTMLHint.utils.isClassExsits(event.attrs, 'tooltip-help');
+            return tagName === 'span' && hasTooltipClass;
+        };
+        var isFocusableElement = function (event) {
```
Comment (rziv) [inline src/rules/accessibility/tooltip-accessibility.js]: move isFocusableElement  to utils

File: src/rules/accessibility/tooltip-accessibility.js [hunk 3]
```diff
@@ -0,0 +1,66 @@
+HTMLHint.addRule({
+    id: 'tooltip-accessibility',
+    description: 'element with binding "addTooltip" should have binding "addDescription"',
```
Comment (rziv) [inline src/rules/accessibility/tooltip-accessibility.js]: whats the addTooltip binding? doesn't appear in the code / in the tests

File: src/rules/accessibility/tooltip-accessibility.js [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+            var tagName = event.tagName;
+            var hasTooltipClass = HTMLHint.utils.isClassExsits(event.attrs, 'tooltip-help');
+            return tagName === 'span' && hasTooltipClass;
+        };
+        var isFocusableElement = function (event) {
+            var focusabaleTagNames = ['input', 'textare', 'select', 'a', 'button'];
+            return focusabaleTagNames.includes(event.tagName);
+        };
+        var getElementDescription = function (event) {
+            var bindings = HTMLHint.utils.getAttributeValue(event.attrs, "data-bind");
```
Comment (rziv) [inline src/rules/accessibility/tooltip-accessibility.js]: make it more generic and expose in utils as function that gets an element and a binding name and returns a binding value (mihjt be empty string) or undefined if this binding doesn't exist

Comment (YahelCh) [inline src/rules/accessibility/tooltip-accessibility.js]: Is there a point to add a test file for utils methods?

**Pull request merged**